android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1221,6 +1221,9 @@ public static Profile getProfile(boolean isIncognito) {
         if (chromeActivity == null) {
             return ProfileManager.getLastUsedRegularProfile(); // Last resort
         }
+        if (!chromeActivity.areTabModelsInitialized()) {
+            return ProfileManager.getLastUsedRegularProfile();
+        }
```

Comment:
Because both return `ProfileManager.getLastUsedRegularProfile();` I would unify this if statement with the one above by simply changing the condition into `if (chromeActivity == null || !chromeActivity.areTabModelsInitialized())`

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/26724/commits/a2a11d86b180ae6b267acd3d1c229e63aba9695c